app/src/index.tsx

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 import ReactDom from "react-dom/client";
 
-import { App } from "./App";
+/**
+ * This is needed to support modulepreload in Vite when using a non HTML custom entrypoint (i.e., this file).
+ * We do this because our index.html is served by the Phoenix server
+ * @see https://vitejs.dev/config/build-options#build-modulepreload
+ */
+import "vite/modulepreload-polyfill";
+
+import App from "./App";
```

Comment:
curious why this needed to be switched to default export?

Reply:
ahh it probably doesn't in the early days i saw something that said that wa causing issues, I don't think it actually was the cause can switch back and see